box2d.pro

```diff
@@ -9,8 +9,13 @@ OBJECTS_DIR = .obj
 
 contains(QT_CONFIG, reduce_exports): CONFIG += hide_symbols
 
-INCLUDEPATH += .
-include(Box2D/box2d.pri)
+# uncomment the line below to compile qml_box2d plugin with Box2D library, installed in OS
+# warning: Box2D library must be already installed in system, for example in Debian it could be done with:
+# sudo apt-get install libbox2d2.3.0 libbox2d-dev
```

Comment:
`libbox2d-dev` will pull in `libbox2d2.3.0` as dependency so it's not necessary to mention it explicitly.